lib/brcobranca/boleto/base.rb

```diff
@@ -156,8 +156,10 @@ def valor_documento
       # @return [Date]
       # @raise [ArgumentError] Caso {#data_documento} esteja em branco.
       def data_vencimento
-        fail ArgumentError, 'data_documento não pode estar em branco.' unless data_documento
+        fail ArgumentError, 'Data Documento não pode estar em branco.' unless data_documento.present?
```

Comment:
Line is too long. [101/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.